ui/app/mainui/activitycenter/views/ActivityNotificationProfileFetching.qml

```diff
@@ -0,0 +1,159 @@
+        property string badgeColor: ""
+    }
+
+    bodyComponent: RowLayout {
+        spacing: 8
+
+        StatusSmartIdenticon {
+            Layout.preferredWidth: 40
+            Layout.preferredHeight: 40
+            Layout.alignment: Qt.AlignTop
+            Layout.leftMargin: Theme.padding
+            Layout.topMargin: 2
+
+            asset {
+                width: 24
+                height: width
+                name: d.assetName
+                color: Theme.palette.primaryColor1
+                bgWidth: 40
+                bgHeight: 40
+                bgColor: d.assetBgColor
+            }
+
+            bridgeBadge.visible: true
+            bridgeBadge.border.width: 2
+            bridgeBadge.color: d.badgeColor
+            bridgeBadge.image.source: Theme.svg(d.badgeName)
+        }
+
+        ColumnLayout {
+            spacing: 2
+            Layout.alignment: Qt.AlignTop
+            Layout.fillWidth: true
+
+            StatusMessageHeader {
+                Layout.fillWidth: true
+                displayNameLabel.text: d.title
+                timestamp: root.notification.timestamp
+            }
+
+            RowLayout {
+                spacing: Theme.padding
+
+                StatusBaseText {
+                    Layout.fillWidth: true
+                    text: d.info
+                    font.italic: true
+                    wrapMode: Text.WordWrap
+                    color: Theme.palette.baseColor1
+                }
+            }
+        }
+    }
+
+    ctaComponent: StatusFlatButton {
+                    size: StatusBaseButton.Size.Small
+                    text: d.ctaText
+                    onClicked: {
+                        root.tryAgainClicked()
+                    }
+                }
+
+    states: [
+        State {
+            when: root.type === ActivityNotificationProfileFetching.FetchingState.Fetching
+            PropertyChanges {
+                target: d
+                info: qsTr("Fetching all data may take some time")
+                badgeName: "dotsLoadings"
+                ctaText: ""
+                badgeColor: Theme.palette.baseColor3
+            }
+        },
+        State {
+            when: root.type === ActivityNotificationProfileFetching.FetchingState.Success
+            PropertyChanges {
+                target: d
+                info: qsTr("Profile details fetched successfully")
+                badgeName: "check"// TODO fix icon it looks bad
+                ctaText: ""
+                badgeColor: Theme.palette.successColor1
+            }
+        },
+        State {
+            when: root.type === ActivityNotificationProfileFetching.FetchingState.PartialFailure
+            PropertyChanges {
+                target: d
+                info: qsTr("Some profile details could not be fetched")
+                badgeName: "exclamation_outline" // TODO fix icon it looks bad
+                ctaText: qsTr("Try again")
```

Comment:
This is the default inside `d`; no need to change it to the same value (similar for other props inside the various `PropertyChanges`)

Reply:
Oops, `""` is actually the default.I switched it in `d`.